Sources/SwiftFusion/Image/Patch.swift

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020 The SwiftFusion Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import TensorFlow
+
+extension Tensor where Scalar == Double {
+  /// Returns the patch of `self` at `region`, interpolating pixels using `interpolation`.
+  ///
+  /// Coordinate system used in `region`: The pixel at `self[j, i, ...]` is the square bounded by
+  /// the corners with coordinates `(i, j)` and `(i + 1, j + 1)`. The center of the pixel is at
+  /// coorindate `(i + 0.5, j + 0.5)`.
```

Comment:
PS, this is the same convention you had, I'm just trying to be more prescriptive about naming conventions and typing :-/

Reply:
Thanks for this detailed explanation!

> I am proposing we include the snippet above in documentation and stick to it

Do you mean put the snippet in this doc comment, or create some standalone doc file like `ImageOperations.md` with this information?

Reply:
I think just copy Frank's comments as Markdown and paste to the comments @marcrasi

Reply:
I put them in `docs/ImageOperations.md`. I think it's useful to have it there because there are methods in multiple different files that need to refer to them.